cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -92,13 +93,32 @@ def plot(self, ax: Optional[plt.Axes] = None, **plot_kwargs: Any) -> plt.Axes:
             fig, ax = plt.subplots(1, 1, figsize=(8, 8))  # pragma: no cover
             ax = cast(plt.Axes, ax)  # pragma: no cover
         ax.set_ylim((0.0, 1.0))  # pragma: no cover
-        ax.plot(self._num_cfds_seq, self._gnd_state_probs, 'ro-', **plot_kwargs)
+        ax.plot(self._num_cfds_seq, self._gnd_state_probs, 'ro', **plot_kwargs)
+        x = np.linspace(self._num_cfds_seq[0], self._num_cfds_seq[-1], 100)
+        fit = self._fit_exponential()
+        ax.plot(x, fit[0][0] * fit[0][2] ** x + fit[0][1], '--k')
         ax.set_xlabel(r"Number of Cliffords")
         ax.set_ylabel('Ground State Probability')
         if show_plot:
             fig.show()
         return ax
 
+    def pauli_error(self):
+        """Returns the Pauli error inferred from randomized benchmarking."""
+        fit = self._fit_exponential()
+        p = fit[0][2]
+        return (1.0 - 1.0 / 4.0) * (1.0 - p)
+
+    def _fit_exponential(self):
+        exp_fit = lambda x, A, B, p: A * p**x + B
+        return curve_fit(
```

Comment:
for readability, could we be more verbose here. also do we acutally need to return everything that `curve_fit` returns? should we just return the optimal parameters?

```py3
def _fit_exponential(self)  -> np.ndarray:
    exp_fit = lambda x, A, B, p: A * p**x + B
    return curve_fit(
            f=exp_fit,
            xdata=self._num_cfds_seq,
            ydata=self._gnd_state_probs,
            p0=[0.5, 0.5, 1.0 - 1e-3],
            bounds=([0, 0.25, 0], [0.5, 0.75, 1]))[0]
```

Reply:
I prefer to return everything since it makes debugging easier if there is a bad fit on a real dataset.

Reply:
sounds reasonable. in that case can you make the function call verbose by adding the name of the parameters
```py3
    return curve_fit(
            f=exp_fit,
            xdata=self._num_cfds_seq,
            ydata=self._gnd_state_probs,
            p0=[0.5, 0.5, 1.0 - 1e-3],
            bounds=([0, 0.25, 0], [0.5, 0.75, 1]))
```

Reply:
What do you mean by a verbose function call?

Reply:
I mean adding the name of the parameters to the function call. so `foo(a=1, b=2, c=3)` instead of `foo(1, 2, 3)`